SparkPost/2web2ui
# SE-269: UX improvements to message events table

### What Changed
 - New Column (subject), truncated with ellipses
- Order of columns rearranged, Event, subject, Recipient, from, timestamp, view details
- Timestamp shows full date after 60 seconds

### How To Test
 - Go to `/reports/message-events`
 -Check that order of columns is correct
- Check that older timestamps show the full timestamp
 - Check that subject truncates if too long.

### To Do

### Notes
 - The timestamp doesn't transition to full timestamp after 1 minute since TimeAgo watches for new time, but we currently don't
